### DISCUSSION TITLE
Logs in a bottom tab (a wider view)

### DISCUSSION THREAD

**Discussion by nathanael-h** [getseabird/seabird#64] [Ideas]
### Summary

Hello, logs are often a little bit long. Could you consider to open them not in the right panel, but in a bottom panel, which could be as wide as the window? 

### Motivation

This would permit to have less wrapped lines.

### Proposal

My suggestion is driven by the habit to see logs in the bottom panel in Lens, and also in the vscode terminal (which is in the bottom by default).

**Comment by nathanael-h** [comment]
Here is how it is in seabird:
![Capture d’écran du 2024-03-06 14-51-48](https://github.com/getseabird/seabird/assets/7300309/9f842a7b-4297-4fd8-b669-54e3ec117d69)
The right panel might be wider a bit.

Here is how it is in let's vscode bottom panel: 
![Capture d’écran du 2024-03-06 14-51-14](https://github.com/getseabird/seabird/assets/7300309/c221f278-6a66-4a5b-9d3c-0dc2484081d4)
The panel being wider, it is more confortable to read logs, espacially when string have fixed lenghts

**Comment by jgillich** [comment]
Check out #71

I have experimented with bottom panels before, but they just trade horizontal for vertical space so I don't think they are a real solution. VS Code is only usable in maximized state (on average size displays).

**Reply by nathanael-h** [reply]
Hello ! Sorry for the delay, I tested the 0.3.2 release, and I can say that the changes you made to the main windows are great: 

- The right panel can be wide enough to have logs and see full lines
- This same right panel to open details about resources is very pleasant to use, I can switch very quickly from one resource to another one
- I can have also the same kind of smooth quick switches thanks to the left column that is always visible, to switch from one kind to another
- The pin mechanism is a big big bonus
👏👏👏
